FlightPlanner/Storage/FlightStorage.cs

```diff
@@ -0,0 +1,155 @@
+                    a.To.City.ToLower().Trim().Contains(userInput)).Select(a => a.To).ToList();
+                return fromAirport.Concat(toAirport).ToList();
+            }
+        }
+
+        public static void Clear()
+        {
+            _id = 0;
+            _flights.Clear();
+        }
+
+        public static bool Exists(AddFlightRequest request)
+        {
+            lock (_lock)
+            {
+                return _flights.Any(f =>
+                    f.Carrier.ToLower().Trim() == request.Carrier.ToLower().Trim() &&
+                    f.DepartureTime == request.DepartureTime &&
+                    f.ArrivalTime == request.ArrivalTime &&
+                    f.From.AirportName.ToLower().Trim() == request.From.AirportName.ToLower().Trim() &&
+                    f.To.AirportName.ToLower().Trim() == request.To.AirportName.ToLower().Trim());
+            }
+        }
+
+        public static bool IsValid(AddFlightRequest request)
+        {
+            lock (_lock)
+            {
+                if (request == null)
+                    return false;
+
+                if (string.IsNullOrEmpty(request.ArrivalTime) ||
+                    string.IsNullOrEmpty(request.Carrier) ||
+                    string.IsNullOrEmpty(request.DepartureTime))
+                    return false;
+
+                if (request.From == null ||
+                    request.To == null)
+                    return false;
+
+                if (string.IsNullOrEmpty(request.From.AirportName) ||
+                    string.IsNullOrEmpty(request.From.City) ||
+                    string.IsNullOrEmpty(request.From.Country))
+                    return false;
+
+                if (string.IsNullOrEmpty(request.To.AirportName) ||
+                    string.IsNullOrEmpty(request.To.City) ||
+                    string.IsNullOrEmpty(request.To.Country))
+                    return false;
+
+                if (request.From.Country.ToLower().Trim() == request.To.Country.ToLower().Trim() &&
+                    request.From.City.ToLower().Trim() == request.To.City.ToLower().Trim() &&
+                    request.From.AirportName.ToLower().Trim() == request.To.AirportName.ToLower().Trim())
+                    return false;
+
+                var arrivalTime = DateTime.Parse(request.ArrivalTime);
+                var departureTime = DateTime.Parse(request.DepartureTime);
+
+                if (arrivalTime <= departureTime)
+                    return false;
+
+                return true;
+            }
+        }
+
+        public static PageResult SearchFlight()
+        {
+            lock (_lock)
+            {
+                return new PageResult(_flights);
+            }
+        }
+
+        public static bool IsValidSearch(SearchFlightRequest request)
+        {
+            lock (_lock)
+            {
+                if (request == null)
+                    return false;
+
+                if (string.IsNullOrEmpty(request.DepartureDate) ||
```

Comment:
šeit var uzreiz return

Reply:
![image](https://user-images.githubusercontent.com/82604527/154506822-cda6f168-bee3-40a4-9eeb-13588554069f.png)

man nepatīk, neuzskatu to par lasāmu. Izvēlos atstāt kā bija